### PR TITLE
Skip crossvit_9_240 TORCH backend in conformance test

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Changes

1) Skip by backend approach is implemented in test conformance
2) crossvit_9_240 is skipped for TORCH backend

### Reason for changes

To make conformance test results consistent

### Related tickets

109655

### Tests

- [ ] Conformance test

### CODE REVIEW

**Review by alexsu52 — APPROVED** [review]
LGTM

File: tests/post_training/model_scope.json [hunk 1]
```diff
@@ -184,6 +184,9 @@
         }
     },
     "crossvit_9_240": {
+        "skipped_by_backends": {
+            "TORCH": "Wrong quantization config, model is validated by TORCH_PTQ backend"
```
Comment (AlexanderDokuchaev) [inline tests/post_training/model_scope.json:188]: How linked config for TORCH and validation on TORCH_PTQ?

And what about another transformer models with same error, like levit_128?
```
RuntimeError: No match has been found among the model operations for the following ignored/target scope definitions:
- ignored_scope: ['{re}.*Embeddings.*', '{re}.*__truediv__*', '{re}.*matmul_1']
 Refer to the original_graph.dot to discover the operations in the model currently visible to NNCF and specify the ignored/target scopes in terms of the names there.
```

And is it planning to fix?

Comment (daniil-lyakhov) [reply tests/post_training/model_scope.json]: Comments are addressed, please check

**Review by AlexanderDokuchaev — APPROVED** [review]

**Comment by daniil-lyakhov** [comment]
One more quick conformance test is no the way

**Comment by daniil-lyakhov** [comment]
Conformance test works as expected, ready for merge

**Pull request merged**